### PR TITLE
Node improvements

### PR DESCRIPTION
**Pull request by skylarmb**
One PR of many, split out for easier reviewing. The overall diff can be seen here: https://github.com/plaid/quickstart/pull/167 

- Docker caching improvements
- implements `.env` file usage with `dotenv` package and uses standard `process.env` calls instead of envvar library
- fixes use of `var` and quotes for object keys
- removes duplicate readme as it has been merged into the main readme in https://github.com/plaid/quickstart/pull/162 / https://github.com/plaid/quickstart/pull/167

### CODE REVIEW

File: node/index.js [hunk 1]
```diff
@@ -1,26 +1,29 @@
...
-var plaid = require('plaid');
-
-var APP_PORT = envvar.number('APP_PORT', 8000);
-var PLAID_CLIENT_ID = envvar.string('PLAID_CLIENT_ID');
-var PLAID_SECRET = envvar.string('PLAID_SECRET');
-var PLAID_ENV = envvar.string('PLAID_ENV', 'sandbox');
+const APP_PORT = process.env.APP_PORT || 8000;
+const PLAID_CLIENT_ID = process.env.PLAID_CLIENT_ID;
+const PLAID_SECRET = process.env.PLAID_SECRET;
+const PLAID_ENV = process.env.PLAID_ENV || "sandbox";
```
Comment (skylarmb) [inline node/index.js:13]: everything below this point is just prettier formatting and cost/let/var changes

File: node/.nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+v14.15.0
```
Comment (cgfarmer4) [inline node/.nvmrc:1]: 🚀

File: node/index.js (hunk 1 above)
Comment (cgfarmer4) [reply node/index.js:13]: I thought we reco'd single quotes for prettier?

File: node/index.js [hunk 3]
```diff
@@ -29,171 +32,179 @@ var PLAID_COUNTRY_CODES = envvar.string('PLAID_COUNTRY_CODES', 'US').split(',');
...
 // Initialize the Plaid client
 // Find your API keys in the Dashboard (https://dashboard.plaid.com/account/keys)
-var client = new plaid.Client({
+const client = new plaid.Client({
   clientID: PLAID_CLIENT_ID,
   secret: PLAID_SECRET,
   env: plaid.environments[PLAID_ENV],
   options: {
-    version: '2019-05-29',
+    version: "2019-05-29"
```
Comment (cgfarmer4) [inline node/index.js:58]: should this be latest?

File: node/index.js (hunk 1 above)
Comment (cgfarmer4) [reply node/index.js:13]: Probably makes sense to commit a `.prettierrc` here if we are going to run it.

File: node/index.js (hunk 3 above)
Comment (cgfarmer4) [reply node/index.js:58]: O I see you're still supporting 6.

**Review by cgfarmer4 — COMMENTED** [review]
Lets add a prettier file for whoever runs this next and I'll stamp.

File: node/index.js (hunk 3 above)
Comment (skylarmb) [reply node/index.js]: updating to latest is a different project alltogether

**Comment by skylarmb** [comment]
@cgfarmer4 addressed your comments

**Review by cgfarmer4 — APPROVED** [review]

**Pull request merged**